docs/resize-event-handling-guide.md

```diff
@@ -0,0 +1,57 @@
+---
+id: resize-event-handling-guide
+title: Handling `resize` event
```

Comment:
Maybe `Responding to resize` is better since it says about the use case rather than implementation?